rskj-core/src/main/java/co/rsk/net/NodeMessageHandler.java

```diff
@@ -145,10 +161,27 @@ private void tryAddMessage(Peer sender, Message message) {
         }
     }
 
+    /**
+     * Responds if a message must be allowed 
+     */
+    private boolean messageIngressControl(Peer sender, Message message) {
+    
+    	AtomicInteger counter = mesagesPerNode.computeIfAbsent(sender.getPeerNodeID(), m -> new AtomicInteger());
+
+    	return (counter.get() < config.getMessageQueueMaxSize());
```

Comment:
could we extract `config.getMessageQueueMaxSize()`'s value into a private field? So that we won't rely on how the value is being extracted internally.